src/io/flutter/utils/HtmlBuilder.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright 2019 The Chromium Authors. All rights reserved.
+ * Use of this source code is governed by a BSD-style license that can be
+ * found in the LICENSE file.
+ */
+package io.flutter.utils;
+
+public class HtmlBuilder {
+
+  public static String span(String style, String contents) {
+    return "<span style = \"" + style + "\"> " + contents + "</span>";
+  }
+
+  public static String span(String contents) {
+    return tag("span", contents);
+  }
+
+  public static String attrs(String attribute, String value) {
```

Comment:
nit: why the s? This looks like a single attribute.

Reply:
Yeah.  It was plural.  Fixing.